srcalc/src/main/java/gov/va/med/srcalc/web/controller/EnterVariablesController.java

```diff
@@ -216,6 +216,18 @@ private boolean bindingResultAlreadyContainsError(final String dynamicKey,
             groupList.add(new PopulatedDisplayGroup(varList, calculation.getPatient()));
         }
         
+        final VariableGroup medicationsGroup = new VariableGroup("Medications", 3);
```

Comment:
Now there is semantic coupling between `EnterVariablesController` and `PopulatedVariableGroup`: both know something about the reference info special cases and must work together to accomplish it. We should move this knowledge to a single location.